llmfoundry/yamls/mosaic_gpt/testing.yaml

```diff
@@ -13,10 +13,11 @@ model:
   d_model: 128
   n_heads: 2
   n_layers: 2
-  mlp_ratio: 4
+  expantion_ratio: 4
```

Comment:
there a bunch more `mlp_ratio` in the yamls (and also this typo)

Reply:
fixed